dsls/sql/src/main/java/org/apache/beam/dsls/sql/transform/BeamAggregationTransforms.java

```diff
@@ -59,13 +59,16 @@
   public static class MergeAggregationRecord extends DoFn<KV<BeamSqlRow, BeamSqlRow>, BeamSqlRow> {
     private BeamSqlRecordType outRecordType;
     private List<String> aggFieldNames;
+    private int windowFieldIdx;
```

Comment:
`windowStartFieldIdx` sounds more clear. Currently there's no field for the index of window in Calcite but maybe possible in future.
 
Would like to keep '-1' here as that's common used when an index is not available.